### DISCUSSION TITLE
Dev Build Verification Errors

### DISCUSSION THREAD

**Discussion by chrpai** [wixtoolset/issues#8165] [WiX Development]
I thought I saw someone post they couldn't build WiX so I decided to revalidate the instructions.  Everything seems good except now I'm getting this error:

```
Installed:
    6 package(s) to packages.config projects
Restore succeeded in 2.0s
    C:\GitHub\wix\src\ext\NetFx\wixlib\netfx.wixproj : error NU1903: Package 'WixToolset.Util.wixext' 0.0.0-build.build.980 has a known high severity vulnerability, https://github.com/advisories/GHSA-jx4p-m4wm-vvjg [C:\GitHub\wix\src\ext\NetFx\NetFx.wixext.sln]

Build failed with errors in 2.5s

```

I assume the core team must be seeing this also.  Is there a trick to get build to know that despite the version 0.0.0 this is a dev build that is newer then the affected production packages?

**Comment by bevanweiss** [comment]
Are you still seeing this @chrpai?
I've successfully run a simple 'devbuild.cmd' (non-release for now) without any breaking errors.

I can see that Rob did commit some fixes to the wix repos which addressed the vulnerabilities raised however.  So perhaps a more recent Nuget package would fix things.  Perhaps try again (if you haven't already).
